docs/user-guide/zwe-init-subcommand-overview.md

```diff
@@ -0,0 +1,202 @@
+
+For more information about `zwe init security`, see [Initializing Zowe security configurations](./initialize-security-configuration.).
+
+
+## Performing APF authorization of load libraries (`zwe init apfauth`)
+
+Zowe contains load modules that require access to make privileged z/OS security manager calls. These load modules are held in two load libraries which must be APF authorized. 
+
+:::info**Required roles:** security administrator
+:::
+
+The command `zwe init apfauth` reads the PDS names for the following load libraries from zowe.yaml and performs the APF authority commands.
+
+* **zowe.setup.dataset.authLoadLib**  
+Specifies the user custom load library, containing the ZWELNCH, ZWESIS01 and ZWESAUX load modules. These are the Zowe launcher, the ZIS cross memory server and the auxiliary server.
+* **zowe.setup.dataset.authPluginLib**
+References the load library for ZIS plugins.
+
+For more information about `zwe init apfauth` see [Performing APF authorization of load libraries](./apf-authorize-load-library).
+
+## Configuring Zowe to use TLS certificates (`zwe init certificate`)
+
+Zowe uses digital certificates for secure, encrypted network communication over Secure Sockets Layer/Transport Layer Security (SSL/TLS) and HTTPS protocols. 
+
+:::info**Required roles:** system programmer, security administrator
+:::
+
+Zowe supports using either file-based (PKCS12) or z/OS key ring-based (when on z/OS) keystores and truststores, and can reuse compatible stores. You can use the `zwe init certificate` command to create keystores and truststores by either generating certificates or by allowing users to import their own compatible certificates.
+
+For more information, see [Configuring certificates](./configure-certificates).
+
+## Creating VSAM caching service datasets (`zwe init vsam`)
+
+Zowe can work in a high availability (HA) configuration where multiple instances of the Zowe launcher are started, either on the same LPAR or different LPARs connected through sysplex distributor. If you are only running a single Zowe instance on a single LPAR you do not need to create a caching service so you may skip this step.
+
+:::info**Required roles:** system programmer
+:::
+
+The command `zwe init vsam` uses the template JCL in `SZWESAMP(ZWECSVSM)` to copy the source template member from `zowe.setup.mvs.hlq.SZWESAMP(ZWECVCSM)` and creates a target JCL member in `zowe.setup.mvs.jcllib(ZWECVSCM)` with values extracted from the `zowe.yaml` file.
+
+For more information about `zwe init vsam`, see [Creating VSAM caching service datasets](./initialize-vsam-dataset).
```

Comment:
Link leads to old VSAM page, should point to section of configure-caching-service-ha